theme/templates/resource-landing-page/modals.html

```diff
@@ -1134,7 +1134,9 @@ <h4 class="modal-title" id="edit-relation"> Edit Relation </h4>
                                         <select class="form-control input-sm select" id="id_relation-{{ forloop.counter0 }}-type" name="relation-{{ forloop.counter0 }}-type">
                                             <option value="">---------</option>
                                             {% for type_value, type_display in relation_source_types %}
-                                                <option value={{ type_value }} {% if relation.type == type_value %}selected="selected"{% endif %}>{{ type_display }}</option>
+                                                {% if type_display != "The content of this resource is hosted by" %}
```

Comment:
Why are relations being changed in this PR?

Reply:
Developers are perhaps most accustomed to having GitHub Issues to review for the functional requirements and use cases for a Pull Request. In this case, a google document has substituted for this step. Please find attached https://docs.google.com/document/d/1Rn72NMEo2QSErzoC6KPItp5bKjRwbqed8GWYDBqlHwg

Reply:
@sblack-usu - there was at least one instance where adding a particular type of reference injected information into the citation automatically. These changes should be removing any instances where relations have any effect on the citation. Once complete - the citation and references should be completely decoupled.